### PR TITLE
Doc / Arrebol worker

### LINKED ISSUE #129: Create Arrebol Worker doc
There are some steps for configuring the worker with Arrebol in the context of using SAPS, for example, mapping the NFS temp storage on the VM for purposes of use related to task processing.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Resolved #129

### CODE REVIEW

**Review by wesleymonte — CHANGES_REQUESTED** [review]
We already have ansible scripts capable of performing these operations automatically, so we just need to indicate the source of these scripts and show the steps to connect to the nfs server. [Take a look.](https://github.com/wesleymonte/worker-deployment)

File: docs/arrebol-worker.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Install and Configure Arrebol worker
+
+The Arrebol worker is a job processing resource that is given one or more tasks to perform making Arrebol manage each of its steps and update important information for its users such as the state of the job, which command is being executed and the exitcode of the commands. The Scheduler component uses Arrebol to send the same task 3 times, each time in the respective phases of the SAPS pipeline (inputdownloading, preprocessing and processing).
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]: It has implementation details described unnecessarily. It is enough to explain that we use Arrebol as a processing service for Jobs and Workers are the resources used by Arrebol.

File: docs/arrebol-worker.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## About the Docker dependency
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]:
```suggestion
## Setting up
```

File: docs/arrebol-worker.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+This configuration is important, as workers are used as Arrebol's labor to perform the work and return information about the execution for use by the service user, for this [access this link to know about the documentation of this step](https://github.com/wesleymonte/worker-deployment).
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]:
```suggestion
To configure your Worker, follow the steps outlined [here](https://github.com/wesleymonte/worker-deployment). When they are successfully completed, proceed to the instructions below).
```

File: docs/arrebol-worker.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  
+## Dependencies
+
+In an apt-based Linux distribution, enter the commands below to install NFS client with the following command:
+
+```bash
+sudo apt-get update
+sudo apt-get install nfs-common
+```
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]: Move that session into the NFS Client

**Review by wesleymonte — CHANGES_REQUESTED** [review]

File: docs/arrebol-worker.md (hunk 4 above)
Comment (thiagoyeds) [reply docs/arrebol-worker.md]: Done

File: docs/arrebol-worker.md (hunk 1 above)
Comment (thiagoyeds) [reply docs/arrebol-worker.md]: Done

File: docs/arrebol-worker.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+sudo mount -t nfs <nfs-temp-storage-ip>:<nfs-temp-storage-path> /nfs
+```
+
+Where:
+- **nfs-temp-storage-ip** is the IP of the VM that contains the NFS temp storage
+- **nfs-temp-storage-path** is the same path used by SAPS for NFS temp storage as mentioned [here](https://github.com/ufcg-lsd/saps-engine/blob/develop/docs/archiver-install.md#temporary-storage)
+
+## Test
+
+### NFS Client
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]: Remove line 33 and update line 31 adding one more hashtag.
`### Test`.

File: docs/arrebol-worker.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+# Setting up Arrebol worker
+
+## Docker
+
+To configure Docker, follow the steps outlined [here](https://github.com/wesleymonte/worker-deployment). When they are successfully completed, proceed to the instructions below).
+
+## NFS Client
+
+### Dependencies
```
Comment (wesleymonte) [inline docs/arrebol-worker.md]: Add a simple explanation that these instructions must be executed on the VM (Worker) to install and configure the NFS Client.

**Review by wesleymonte — CHANGES_REQUESTED** [review]

File: docs/arrebol-worker.md (hunk 5 above)
Comment (thiagoyeds) [reply docs/arrebol-worker.md]: Done

File: docs/arrebol-worker.md (hunk 6 above)
Comment (thiagoyeds) [reply docs/arrebol-worker.md]: Done

**Review by wesleymonte — APPROVED** [review]
Nice Job @thiagoyeds !

**Review by thiagomanel — APPROVED** [review]
I've edited the text directly. it lgtm now

**Pull request merged**